extension/page-detect.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Yeah, that's intended. The plunker is just illustrating that setting `use strict` in `script.js` doesn't impact code in `script2.js`. We should probably check if that's different for content scripts, though.